c/mptcphello.c

```diff
@@ -5,34 +5,52 @@
 #include <sys/socket.h>
 #include <stdbool.h>
 #include <netinet/in.h>
+#include <errno.h>
 
+// IPPROTO_MPTCP is defined in <netinet/in.h> on recent kernels only
```

Comment:
(it depends on the libc version in fact, not the kernel in fact but that's probably a detail)